public/app/plugins/datasource/prometheus/datasource.test.ts

```diff
@@ -1601,9 +1601,24 @@ describe('PrometheusDatasource', () => {
           text: expectedRangeSecond * 1000,
           value: expectedRangeSecond * 1000,
         },
+        __rate_interval: {
+          text: '75s',
+          value: '75s',
+        },
       });
     });
   });
+
+  describe('The __rate_interval variable', () => {
+    it('should be 4 times the scrape interval if interval + scrape interval is lower', () => {
+      const { __rate_interval } = ds.getRateIntervalScopedVariable(15, 15);
```

Comment:
Perhaps use a "weirder" value here than 15s? With 15s, the expected result is 60s, which happens to be the same as if 0s is used.

Reply:
Done